.travis.yml

```diff
@@ -87,7 +87,7 @@ script:
 - if [ "$MPI" ]; then
     testflo . --coverage --coverpkg openmdao -i;
   else
-    testflo . --coverage --coverpkg openmdao;
+    testflo . --coverage --coverpkg openmdao -n 1;
```

Comment:
not actually needed for pypi, but something @naylor-b asked for in general for CI, so I threw it in here.